tests/framework-tests/preferences/js/PrefsEditorTests.js

```diff
@@ -68,9 +66,20 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
                         }
                     }
                 },
+                saveCalled: false,
+                prefsEditorRefreshed: 0,
                 listeners: {
-                    onSave: function () {
-                        saveCalled = true;
+                    "onSave.called": {
+                        listener: function (that) {
+                            that.options.saveCalled = true;
```

Comment:
See below - could you try to harmonise these changes with the ones in my branch?